src/vulnpy/trigger/__init__.py

```diff
@@ -9,6 +9,7 @@
 DATA = {
     "cmdi": "echo attack",
     "deserialization": "csubprocess\ncheck_output\n(S'ls'\ntR.",
+    "pt": "../../etc/passwd",
```

Comment:
Can you change this to something more descriptive? e.g `patch_traversal` 

Not a big deal just a preference.

Reply:
This has to match everywhere else where I went with `pt` instead of `path_traversal`. The main motivation was to keep public-API imports and URLs short, since I thought `/vulnpy/pt/do_open` looked really nice.

With your blessing, I'd prefer to leave this as-is